studentInfoOGB.py

```diff
@@ -0,0 +1,96 @@
+import random
+
+firstName = ["jay", "jim", "roy", "axel", "billy", "charlie", "jax", "gina", "paul",
+"ringo", "ally", "nicky", "cam", "ari", "trudie", "cal", "carl", "lady", "lauren",
+"ichabod", "arthur", "ashley", "drake", "kim", "julio", "lorraine", "floyd", "janet",
+"lydia", "charles", "pedro", "bradley"]
+
+lastName = ["blake", "holt", "meilijson", "brannon", "fieser", "line", "hellman",
+"lyndon", "wendell", "corr", "fricano", "carrigan", "giddings", "piesse", "charlwood",
+"thaler", "ambrose", "hamer", "schweig", "geiger", "kinder", "rogantin", "chevallier"
+"wang", "tiedemann", "cohn", "digel", "reheem", "dzieciololowski", "callanan",
+"lie", "oden"]
+
+def main():
+  students = [
+    Student("Larsson", 37),
+    Student("BonJovi", 55),
+  ]
+
+  printHeader()
+  selection = int(getUserSelection())
+  if selection == 0:
+    printStudentsByAge(students)
+  elif selection == 1:
+    printStudentsByLName(students)
+  elif selection == 2:
+    printStudentsByFName(students)
+  else:
+    print ("SELECTION NOT RECOGNIZED")
```

Comment:
For your function calls you seem to not leave any space between the function name and the parentheses.

I'm not sure if you're using Python 2.7 or Python 3.x, since your code is compatible with both.

If you were targetting Python 2.7, then since `print` isn't a function* I'd say it's fair to have an extra space. Then again you are including parentheses, treating it as if it was a function.

* `print` is more of a statement, as in `print "Hello World"` would function as expected, but in Python 3.x it was changed to be a function to be compatible with the function grammar.